dropshot/src/handler.rs

```diff
@@ -67,8 +67,10 @@ use std::marker::PhantomData;
 use std::num::NonZeroU32;
 use std::sync::Arc;
 
+pub type ResponseBody = Body;
```

Comment:
the use of this confused me; what's the idea here?

Reply:
This type alias can likely be completely ripped out, it was one of the first things I did to address the mountain of compiler errors (it was `type ResponseBody = BoxBody<...>` originally).